src/Plugin/WebformElement/MineOrganisationsData.php

```diff
@@ -643,9 +649,18 @@ private function getSearchResult(array &$element, string $query): array {
    */
   private function getSearchUserIds(string $query): array {
 
-    $brugere = $this->organisationHelper->searchBruger($query);
+    try {
+      $brugere = $this->organisationHelper->searchBruger($query);
+    }
+    catch (ApiException $e) {
+      $brugere = NULL;
+    }
+
+    if (NULL === $brugere) {
+      $this->messenger()->addMessage($this->t('Could not fetch organisation data. Contact form owner.'));
+    }
 
-    return array_map(static fn($value) => $value['id'], $brugere);
+    return array_map(static fn($value) => $value['id'], $brugere ?? []);
```

Comment:
https://www.php.net/manual/en/function.array-column.php is your friend.

Reply:
👍